### PR TITLE
1081 `POST /update-all` filtered by patientId

### PR DESCRIPTION
**Pull request by leite08**
Ref: metriport/metriport-internal#1181

### Dependencies

- Upstream: none
- Downstream:
   - https://github.com/metriport/metriport-internal/pull/1183
   - https://github.com/metriport/metriport/pull/1019

### Description

- POST /update-all filtered by patientId
- bulk-insert better phone normalization

### Release Plan

- nothing special

### CODE REVIEW

File: packages/api/src/routes/medical/internal-patient.ts [hunk 1]
```diff
@@ -52,10 +52,16 @@ router.post(
...
+    const patientIds = patientIdsRaw?.split(",") ?? [];
+
     const facilities = await getFacilities({ cxId });
     let failedUpdateCount = 0;
     for (const facility of facilities) {
-      const patients = await getPatients({ cxId, facilityId: facility.id });
+      const allPatients = await getPatients({ cxId, facilityId: facility.id });
+      const patients = patientIds.length
+        ? allPatients.filter(p => patientIds.includes(p.id))
+        : allPatients;
```
Comment (Goncharo) [inline packages/api/src/routes/medical/internal-patient.ts:64]: nit: doing the filter in the query would be more efficient than in memory - especially for cxs with large volumes of patients

**Review by Goncharo — APPROVED** [review]
Code looks fine - nit to consider with patient filtering

File: packages/api/src/routes/medical/internal-patient.ts (hunk 1 above)
Comment (leite08) [reply packages/api/src/routes/medical/internal-patient.ts:64]: oh yeah, 💯 

I just didn't want to change much given the time available, but I agree, considering this is something that's being resource intensive its worth doing right

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**